README.md

```diff
@@ -235,27 +253,27 @@ We have a few known features we want to take on from here while we remain open f
 If you would like to propose a feature, please [open an Issue](https://github.com/intelsdi-x/snap/issues)) that includes RFC in it (for [request for comments](https://en.wikipedia.org/wiki/Request_for_Comments)).
 
 ## Community Support
-This repository is one of **many** projects in the **snap framework**. Discuss your questions about snap by reaching out to us:
+This repository is one of **many** projects in the **Snap framework**. Discuss your questions about snap by reaching out to us:
 
 * Through GitHub Issues. Issues is our home for **all** needs: Q&A on everything - installation, request for events, integrations, bug issues, futures. [Open up an Issue](https://github.com/intelsdi-x/snap/issues) and know there's no wrong question for us.
 * We also have a [Gitter channel](https://gitter.im/intelsdi-x/snap) opened up on this repository that threads directly into our engineering team Slack (thanks to [Sameroom](https://sameroom.io/)).
-* Submit a blog post on your use of snap to [our Medium.com publication](https://medium.com/intel-sdi)
+* Submit a blog post on your use of Snap to [our Medium.com publication](https://medium.com/intel-sdi)
 
 The full project lives here, at http://github.com/intelsdi-x/snap.
 
 ## Contributing
-We encourage contribution from the community. **snap** needs:
+We encourage contribution from the community. Snap needs:
 
 * _Feedback_: try it and tell us about it through issues, blog posts or Twitter
 * _Contributors_: We need plugins, schedules, testing, and more
-* _Integrations_: **snap** can feasibly publish to almost any destination by [authoring a plugin](#author-a-plugin)
+* _Integrations_: Snap can collect from and publish to almost anything by [authoring a plugin](#author-a-plugin)
```

Comment:
I don't understand this comment. We collect and publish through plugins and people authoring them.